WalletWasabi.Fluent/ViewModels/Wallets/WalletSettingsViewModel.cs

```diff
@@ -19,10 +18,13 @@ namespace WalletWasabi.Fluent.ViewModels.Wallets;
 	Searchable = false)]
 public partial class WalletSettingsViewModel : RoutableViewModel
 {
+	private readonly IWalletModel _wallet;
 	[AutoNotify] private bool _preferPsbtWorkflow;
+	[AutoNotify] private string _walletName;
```

Comment:
null warning

Reply:
I no longer get this warning. Fixed by previous changes, maybe?